scripts/rename.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash 
+
+set -e 
+
+for i  in $(ls [0-9][0-9][0-9][0-9].yaml)
+do
+  KIND=$(yq .kind $i | tr -d '"');
+  NAME=$(yq .metadata.name $i | tr -d '"');
```

Comment:
@zvonkok the PR mostly looks good to me aside from the nits Jiri mentioned. I tried using the yq from https://github.com/mikefarah/yq/ and the following changes were needed. (There are several ways to install it in the README, not sure if you want to add one to the Makefile to replace the pip install)

```suggestion
  KIND=$(yq r $i kind | tr -d '"');
  NAME=$(yq r $i metadata.name | tr -d '"');
```

Reply:
Reverted to bash. Lets keep this in mind if we need sophisticated yaml manipulation/reading.